sdk/websites/Azure.ResourceManager.AppService/src/Customization/Models/AppServiceTableStorageApplicationLogsConfig.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppService.Models
+{
+    public partial class AppServiceTableStorageApplicationLogsConfig
+    {
+        /// <summary> Uri of the resource. </summary>
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+        [ObsoleteAttribute("This property has been replaced by ResourceUriString", false)]
+        public System.Uri SasUri { get; set; }
+
+        public AppServiceTableStorageApplicationLogsConfig(System.Uri SasUri)
```

Comment:
Let me double confirm - this property never works right?
if so, since the Uri property is obsoleted, we should also add EBNever and Obsolete to this public ctor as well.